timemachine/datasets/utils.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+from typing import List
+
+from rdkit import Chem
+
+import timemachine
+
+PATH_TO_FREESOLV = str(Path(timemachine.__file__).parent / "datasets/freesolv/freesolv.sdf")
```

Comment:
tangential -- if we add `datasets` to `package_data` in `setup.py`, maybe this could eventually be something like
```python
with importlib.resources.path("timemachine", "datasets") as path:
    PATH_TO_FREESOLV = path / "freesolv" / "freesolv.sdf"
    # ...
```
https://docs.python.org/3.8/library/importlib.html?highlight=importlib#importlib.resources.path

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/619/commits/0c434fe4d71fec2f8226e45f22edac5cd413f853